x/distribution/client/cli/query.go

```diff
@@ -154,8 +154,8 @@ func GetCmdQueryValidatorSlashes(queryRoute string, cdc *codec.Codec) *cobra.Com
 func GetCmdQueryDelegatorRewards(queryRoute string, cdc *codec.Codec) *cobra.Command {
 	return &cobra.Command{
 		Use:   "rewards [delegator] [validator]",
-		Args:  cobra.ExactArgs(2),
-		Short: "Query distribution delegator rewards",
+		Args:  cobra.RangeArgs(1, 2),
```

Comment:
Like this approach a lot. I think we could probably refactor a couple of other CLI commands to also exhibit this behavior.